geth/common/types.go

```diff
@@ -257,26 +257,26 @@ type JailCell interface {
 	// Call an arbitrary JS function by name and args.
 	Call(item string, this interface{}, args ...interface{}) (otto.Value, error)
 	// Stop stops background execution of cell.
-	Stop()
+	Stop() error
 }
 
 // JailManager defines methods for managing jailed environments
 type JailManager interface {
-	// Parse creates a new jail cell context, with the given chatID as identifier.
-	// New context executes provided JavaScript code, right after the initialization.
-	Parse(chatID, js string) string
-
 	// Call executes given JavaScript function w/i a jail cell context identified by the chatID.
 	Call(chatID, this, args string) string
 
-	// NewCell initializes and returns a new jail cell.
-	NewCell(chatID string) (JailCell, error)
+	// CreateCell creates a new jail cell.
+	CreateCell(chatID string) (JailCell, error)
+
+	// CreateAndInitCell creates a new jail cell and initialize it
+	// with web3 and other handlers.
+	CreateAndInitCell(chatID string, code ...string) string
 
 	// Cell returns an existing instance of JailCell.
-	Cell(chatID string) (JailCell, error)
+	GetCell(chatID string) (JailCell, error)
```

Comment:
Why renaming that? It's a general recommendation for getter from Effective Go: https://golang.org/doc/effective_go.html#Getters
It also reads well as a sentence: `s.Jail.Cell(chatID)`. Instead of `s.Jail.GetCell(chatID)`.

Reply:
Agree, I will change that.